bundle.yml

```diff
@@ -50,6 +50,14 @@ options :
         help  : Frontend parser to use for Loki transformations
         cmake : LOKI_FRONTEND={{value}}
 
+    - with-gpu :
+        help  : Enable GPU kernels
+        cmake : >
+            ENABLE_CLOUDSC_GPU_SCC=ON
```

Comment:
I am pretty sure none of these have any effect, as they are not defined in the CMakeLists.txt of the Loki variant. In fact, since we have no non-loki GPU configurations in this dwarf, this flag here seems redundant.

Having said that, the fact that the GPU compilation toolchain "just picks up" GPU compiler options is not ideal. However, properly configuring the GPU-specific build options should probably happen in a separate follow-up PR. For now we can probably just remove this?